contrib/pyzfs/setup.py

```diff
@@ -28,8 +29,6 @@
         "Development Status :: 4 - Beta",
         "Intended Audience :: Developers",
         "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python :: 2 :: Only",
-        "Programming Language :: Python :: 2.7",
```

Comment:
Please add the correct classifiers to indicate Python 2 and Python 3 compatibility: https://pypi.python.org/pypi?%3Aaction=list_classifiers

Reply:
It wasn't clear to me which classifiers to use.  Should all applicable ones be added?  i.e.

```
"Programming Language :: Python :: 2.7",
"Programming Language :: Python :: 3.4",
"Programming Language :: Python :: 3.5",
"Programming Language :: Python :: 3.6",
"Programming Language :: Python :: 3.7",
```

Reply:
I'd just set `:: 2`, `:: 3`, and the major ones we're testing with (`:: 2.7`, `:: 3.6`, `:: 3.7`).

Reply:
Done, I kept them all since I found every one of these versions in use by the bots.